public/index.html

```diff
@@ -34,26 +38,55 @@ <h4 class="col-xs-12">File History {{ history.visibility.states }}</h4>
                   <a href ng-click="history.remove(edit)" class="btn btn-danger" title="">x</a>
                 </div>
 
-	        <div class="col-xs-12">
+                <div class="col-xs-12">
                   <div hljs source="edit.content">
                     <textarea></textarea>
-  	          </div>
-	        </div>
+                  </div>
+                </div>
 
               </div>
 
             </div><!-- /line -->
 
           </div>
+
           <div class="col-xs-12 alert alert-info"
             ng-if="history.edits.length === 0"
             class="animate">
             ... aucun historique... mais que fait le prof !
           </div>
+
         </div>
-      </div><!-- / -->
+
+      </div><!-- /history -->
+
     </div><!-- /container -->
 
+
+    <!-- DEBUT HISTORIQUE PERSISTENCE -->
+    <div ng-controller="HistoryPersistenceCtrl as historyPersistenceC">
+      <div ng-repeat="historyP in historyPersistenceC.historyP" class="animate">
```

Comment:
`historyP in historyPersistenceC.historyP` -> `history in historyPersistence.history`